src/Employee.java

```diff
@@ -1,9 +1,11 @@
 public class Employee {
+
+
     private String fullName;
```

Comment:
it is better to use fullNmae separately like first name, last name and patronymic